tests/DoctrineMongoODMModuleTest/Doctrine/CliTest.php

```diff
@@ -26,7 +34,7 @@
  * @link    http://www.doctrine-project.org/
  * @author  Adam Homsi <adam.homsi@gmail.com>
  */
-class CliTest extends PHPUnit_Framework_TestCase
+final class CliTest extends \PHPUnit_Framework_TestCase
```

Comment:
@alcaeus this NS is not public, nor is shipped with the package (it's [dev-only](https://github.com/doctrine/DoctrineMongoODMModule/blob/6f7f00f778fa4a03d28831c019992a32729f9cc1/composer.json#L69-L73), as @Bilge said)